src/test/java/com/wikia/webdriver/common/core/url/FandomUrlBuilder.java

```diff
@@ -0,0 +1,49 @@
+package com.wikia.webdriver.common.core.url;
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+
+import org.apache.commons.lang.StringUtils;
+
+public class FandomUrlBuilder {
+
+  final static String FANDOM_URL = "fandom.wikia.com";
+  final static String ARTICLE_PATH = "articles";
+  public final static String ENV_PROD = "prod";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)